sdk/python/core/docsgen/delete_operation.rst

```diff
@@ -0,0 +1,55 @@
+Delete operation
+****************
+This document explains how to use YDK object to delete nodes. Examples below use `ydktest.json <https://raw.githubusercontent.com/CiscoDevNet/ydk-gen/master/profiles/test/ydktest.json>`_ profile file to generate YDK test package.
```

Comment:
How about something like:

"how to use YDK `delete` operation to delete nodes."